docs/conf.py

```diff
@@ -0,0 +1,59 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+
+import os
+import sys
+
+import autoapi
+
+# Define path to the code to be documented **relative to where conf.py (this file) is kept**
+sys.path.insert(0, os.path.abspath('../src/'))
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'lsstseries'
+copyright = '2023, Doug Branton'
+author = 'Doug Branton'
```

Comment:
Are multiple authors supported here? Would be good to have Neven here if possible.

Reply:
Yeah, the template only adds one, but you should be able to add more. Pretty sure that the author and copyright info are just strings. So you should be able to do:
`author = 'Doug Branton, Neven Caplar'`